src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -550,3 +599,131 @@ export const maskUidOnLastField = (
   )
   return maskedAutoReplyData
 }
+
+/**
+ * Function to extract information for email json field from response
+ * Json field is used for data collation tool
+ */
+const getDataCollationFormattedResponse = (
+  response: ResponseFormattedForEmail,
+): EmailDataCollationToolField | undefined => {
+  const { answer, fieldType } = response
+  // Headers are excluded from JSON data
+  if (fieldType !== BasicField.Section) {
+    const dataCollationData = {
+      question: getJsonPrefixedQuestion(response),
+      answer,
+    }
+    return dataCollationData
```

Comment:
Nitpicky, but abhor state when unnecessary
```suggestion
    return {
      question: getJsonPrefixedQuestion(response),
      answer,
    }
```